proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -296,31 +297,7 @@ message MsgSell {
   string owner = 1;
 
   // orders are the sell orders being created.
-  repeated Order orders = 2;
-
-  // Order is the content of a new sell order.
-  message Order {
-
-    // batch_denom is the credit batch being sold.
-    string batch_denom = 1;
-
-    // quantity is the quantity of credits being sold from this batch. If it is
-    // less then the balance of credits the owner has available at the time this
-    // sell order is matched, the quantity will be adjusted downwards to the
-    // owner's balance. However, if the balance of credits is less than this
-    // quantity at the time the sell order is created, the operation will fail.
-    string quantity = 2;
-
-    // ask_price is the price the seller is asking for each unit of the
-    // batch_denom. Each credit unit of the batch will be sold for at least the
-    // ask_price or more.
-    cosmos.base.v1beta1.Coin ask_price = 3;
-
-    // disable_auto_retire disables auto-retirement of credits which allows a
-    // buyer to disable auto-retirement in their buy order enabling them to
-    // resell the credits to another buyer.
-    bool disable_auto_retire = 4;
-  }
+  repeated SellOrder orders = 2;
```

Comment:
I'm not sure if we should remove `Order` and replace it with `SellOrder`. `SellOrder` includes `owner`, which is also included in `MsgSell` and we should avoid duplicating the `owner` field. Thoughts?

Reply:
yeah i'm not sure what would be the best path forward here. imo i think the big sacrifice is having to manage two structures (if we update SellOrder down the road, we're gonna have to remember to update Order as well). the downside to having both is message size + clunky api consuming, devs will have to embed address in each repeated Sell/BuyOrder

Reply:
talked with Cory, decided its best to have the duplicate structure. i'll revert the changes and put the Order message back in